core/variables.js

```diff
@@ -106,8 +106,8 @@ Blockly.Variables.allDeveloperVariables = function(workspace) {
   var hash = {};
   for (var i = 0; i < blocks.length; i++) {
     var block = blocks[i];
-    if (block.getDeveloperVars) {
-      var devVars = block.getDeveloperVars();
+    if (block.getDeveloperVariables) {
```

Comment:
Changing this without some backward compatible support may break existing blocks, and will fail the generator tests.